### DISCUSSION TITLE
How to use edge attributes in GATConv?

### DISCUSSION THREAD

**Discussion by arake001** [pyg-team/pytorch_geometric#4354] [Q&A] [answered]
I have been trying to use edge_attributes in GATConv but have some issues. Can you give an example?

**Comment by arake001** [comment, marked as answer]
I solved the issue. Thank You!

**Comment by simmons7625** [comment]
how? I have same issue. I need some examples of codes.

**Reply by rusty1s** [reply]
See https://github.com/pyg-team/pytorch_geometric/blob/master/test/nn/conv/test_gat_conv.py#L111.
